casepro/statistics/migrations/0003_auto_20160701_1412.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('statistics', '0002_populate_reply_counts'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='dailycount',
+            name='count',
+            field=models.IntegerField(),
```

Comment:
This was a `PositiveIntegerField` but now we need to record a message losing a label as a negative incremental count